### ISSUE TITLE
UI not showing experiment results

### ISSUE THREAD

**Issue by texasmichelle** [kubeflow/katib#916]
/kind feature

**Describe the solution you'd like**
It would be useful to have more than a blank screen in the Katib UI for PyTorch jobs.
I ran the [PyTorch example](https://github.com/kubeflow/katib/blob/master/examples/v1alpha2/pytorchjob-example.yaml) on a fresh Kubeflow cluster created using [C2D](https://deploy.kubeflow.cloud/#/deploy) with v0.6.2.
It appears to run fine since the containers run to completion and logs look normal, but no metrics appear in the UI after everything completes. It would be nice to have a graph showing results, or at least a message about metric rendering being under construction. A blank screen makes me wonder if the UI is even working.

**Comment by gaocegege** [comment]
Sounds reasonable

**Comment by texasmichelle** [comment]
I thought this issue was limited to PyTorch, but it appears to be broader. I just tried running [tfjob-example](https://github.com/kubeflow/katib/blob/master/examples/v1alpha2/tfjob-example.yaml) as well and am not seeing any visuals in the UI. Perhaps it's something not configured properly, but I'm using the default C2D installation, which runs this image:
gcr.io/kubeflow-images-public/katib/v1alpha2/katib-controller:v0.6.0-rc.0

As with the pytorch example, I can run the experiment successfully and view pod logs, but no information shows up in the UI beyond the experiment name.

**Comment by texasmichelle** [comment]
/kind bug

It's probably something with the configuration, but we should make sure that the default installation picks up components that work together properly.

**Comment by gaocegege** [comment]
@texasmichelle Can you show us the screenshot of UI?

**Comment by wjddyd66** [comment]
I have Same issue.

I run hyperparameter tunning example) tfjob-example
Kubeflow Version: v0.7.0
kubernetes Version: v1.15.0

Result CLI
```code
$ kubectl get experiment -n kubeflow

NAME            STATUS      AGE
tfjob-example   Succeeded   24m
```

But UI is
![image](https://user-images.githubusercontent.com/37298751/69219971-5919b880-0bb7-11ea-9df9-c699465631c7.png)

And Result is
![image](https://user-images.githubusercontent.com/37298751/69220019-68990180-0bb7-11ea-910d-9b06ec1f4287.png)

**Comment by wjddyd66** [comment]
@texasmichelle 

Hey I solved This Problem

I think Your Server in CLI Mode(Ubuntu, Linux)

I Run same Example in GUI Mode(Ubuntu, Linux)

![image](https://user-images.githubusercontent.com/37298751/69319133-f2ad9c80-0c81-11ea-896d-c27151e9c1c3.png)

**It's Work**

**Comment by andreyvelich** [comment]
@texasmichelle 
We updated Katib UI with Trial status and Experiment view.
Should we close this issue?
